README.md

```diff
@@ -48,5 +85,18 @@ This Job will than be scheduled if ressoucces are avaiable.
 If it not scheduled within the timeout the job is canceled to not block ressourcess that are no longer needed.
 To be able to run the runner with the correct enviorment cluster specific bash file needs to be created.
 
+# Cleanup
+
+On some clusters TMPDIR is cleaned per Job on others not.
+This is why TMPDIR is changed to one based on the Job ID and deleted after SIGUSR1 is send by SLURM this is Configured by default in Flame SLurm to 30 Seconds before kill.
+
+# Long running Jobs
+
+If your Job time is Limited Slurm will kill the Job with the Flame runner.
+There are no mechanisims in place to not use the runner if time is about to run out.
+
+This would be a well appriciated contribution!
+
+Be aware that you might loose Data because of this.
```

Comment:
```suggestion
If your job time is limited, Slurm will kill the Job while it is running. There are no mechanisms at the moment to not use the runner if time is about to run out. This would be a well appreciated contribution! Be aware that you might lose data because of this.
```